wagtail/admin/templates/wagtailadmin/shared/animated_logo.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import TemplatePattern from '../../../../../client/storybook/TemplatePattern';
+
+import template from './animated_logo.html';
+
+export default {
+  parameters: {
+    docs: {
+      source: { code: template },
+    },
+  },
```

Comment:
I’ve added this inconsistently. If people find it useful I’d like to do it systematically with a utils-style function to reduce the boilerplate needed.